### PR TITLE
DB-5713: Search Term Persistence

### PR DESCRIPTION
**Pull request by mitchellmarkoff**
## What changes were made?
- The search term entered into the header persists on the search results page.

## Where were the changes made?
`cli > templates > next-drupal-search-api-addon`

## How have the changes been tested?
Search term observed to persist during local development.

## Additional information
As mentioned in the ticket this state persistence needed to be 'faked' since we have two `SearchInput` components, one in `/pages` and one in `/app` 

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

**Review by CobyPear — CHANGES_REQUESTED** [review]
This seems to be working fine if I remove the catchall route, otherwise when I try to search for a query I see "No content found 🏜️"

**Comment by mitchellmarkoff** [comment]
Made this a minor bump because this PR does add SOME functionality. However, since this is a very small amount of work I could see a case for this to be a patch.

File: .changeset/swift-dogs-care.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'create-pantheon-decoupled-kit': minor
+---
+
+[next-drupal] User entered search term persists on search results page in the
```
Comment (CobyPear) [inline .changeset/swift-dogs-care.md:5]:
```suggestion
[next-drupal-search-api-addon] User entered search term persists on search results page in the
```
More specificity with the tag here so we can easily tell which generators were changed at a glance of the changelog

**Review by CobyPear — CHANGES_REQUESTED** [review]
LGTM, just one small change requested on the changeset.

**Review by CobyPear — APPROVED** [review]

**Pull request merged**